Sources/SwiftInspectorCore/Tests/InitializerAnalyzerSpec.swift

```diff
@@ -157,25 +157,6 @@ final class InitializerAnalyzerSpec: QuickSpec {
             ]
           }
         }
-
-        context("with a closure") {
```

Comment:
@fdiaz adding this test seemed like a mistake since it is testing a tuple, just like the one above. Removing in an effort to be helpful but lmk if I missed something.

Reply:
(I made this comment on the PR too 👍 )